cli/src/tests/firstTest.ts

```diff
@@ -0,0 +1,8 @@
+const referee = require("@sinonjs/referee")
+const assert = referee.assert;

```

Comment:
This is using ES5 require syntax, should be converted to a module like this:

```
import referee from '@sinonjs/referee';
const assert = referee.assert;
```

Or could probably reduce to a one-liner like this:

```
import { assert } from '@sinonjs/referee'
```